facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -328,10 +328,10 @@ trait FapiFrontPress extends EmailFrontPress with GuLogging {
         case "nav/list" => storyCountTotal
         // scrollable feature containers are capped at 3 stories
         case "scrollable/feature" => 3
-        // scrollable highlights and medium containers are capped at 6 stories
-        case "scrollable/highlights" | "scrollable/medium" => 6
-        // scrollable/small containers are capped at 8 stories
-        case "scrollable/small" => 8
+        // scrollable highlights  are capped at 6 stories
```

Comment:
There looks to be an extra space in here. Might be worth adding in the word "containers" as well.

```suggestion
        // scrollable highlights containers are capped at 6 stories
```

Reply:
Ah, good catch, thanks!